### PR TITLE
Updated docs core-api section

### PR DESCRIPTION
**Pull request by ksyhoo**

### CODE REVIEW

File: docs/docs/building-sockets/core-api.md [hunk 1]
```diff
@@ -1,10 +1,12 @@
 # Using Core API library
 
 The Core API library should be used in the **Syncano Sockets** (inside the scripts powering the Syncano Sockets) to communicate with the **Syncano Core Services** and 3rd party integrated platforms. Syncano provides various Core Services:
-- **Database (db)** - NoSQL database to store your application data
+- **Database (data)** - NoSQL database to store your application data
 - **Users Management (users)** - service to store and manage users and groups of your application
 - **Event Loop (events)** - service to emit events which can be caught by any Socket
 - **Realtime Channels (channels)** - implement publish/subscribe model for realtime communication
+- **Response method (response)** - lets you send response from Socket
```
Comment (mkucharz) [inline docs/docs/building-sockets/core-api.md]: Those are not actually CORE services, those are just helpers to use inside a Socket. Lets make here two lists (note changes in description also):

Something like that:

Syncano provides various Core Services and Core library contains an interface to them: 
- **Database (data)** - NoSQL database to store your application data
- **Users Management (users)** - service to store and manage users and groups of your application
- **Event Loop (events)** - service to emit events which can be caught by any Socket
- **Realtime Channels (channels)** - implement publish/subscribe model for realtime communication

Library also contains number of helpers for working with a scripts:
- **Response method (response)** - lets you send custom response from the script
- **Logging (logger)** - logger which helps with debugging scripts and measuring time of execution

File: docs/docs/building-sockets/core-api.md [hunk 2]
```diff
@@ -168,6 +189,8 @@ response
 
 ### Logging
 
+Easy way to debugg your script code. Think of it as your console.log but for your socket.
```
Comment (mkucharz) [inline docs/docs/building-sockets/core-api.md]: console.log -> \`console.log\`

Comment (mkucharz) [reply docs/docs/building-sockets/core-api.md]: Typo in *debugg*

Comment (mkucharz) [reply docs/docs/building-sockets/core-api.md]: `Socket` with capital letter

File: docs/docs/building-sockets/core-api.md [hunk 3]
```diff
@@ -118,25 +126,37 @@ event.emit('my_signal', {dummyKey: 'dummy_value'})
   })
 ```
 
+Catch `events` by subscribing to a socket emmiting an event.
```
Comment (mkucharz) [inline docs/docs/building-sockets/core-api.md]: `Socket` (capital letter)

File: docs/docs/building-sockets/core-api.md [hunk 4]
```diff
@@ -110,6 +116,8 @@ users
 
 ### Events (events)
 
+Publish global `events` to which other sockets can subscribe.
```
Comment (mkucharz) [inline docs/docs/building-sockets/core-api.md]: sockets -> Sockets

File: docs/docs/building-sockets/core-api.md [hunk 5]
```diff
@@ -116,25 +126,37 @@ event.emit('my_signal', {dummyKey: 'dummy_value'})
...
-```js
-const latestTags = await socket.get('tags/list', { sort: 'latest' })
-const createdTag = await socket.post('tags/create', { name: 'nature' })
-```
-
 ## Utils
 
 ### Response
 
+Response lets you send custom responses from sockets.
```
Comment (mkucharz) [inline docs/docs/building-sockets/core-api.md]: Sockets.

File: docs/docs/building-sockets/core-api.md [hunk 6]
```diff
@@ -1,12 +1,18 @@
-# Using Core API library
+# Using Core library
 
-The Core API library should be used in the **Syncano Sockets** (inside the scripts powering the Syncano Sockets) to communicate with the **Syncano Core Services** and 3rd party integrated platforms. Syncano provides various Core Services:
-- **Database (db)** - NoSQL database to store your application data
+Syncano provides various Core Services and Core library contains an interface to intereact with them:
```
Comment (mattlsp) [inline docs/docs/building-sockets/core-api.md]: should be "interact"

File: docs/docs/building-sockets/core-api.md [hunk 7]
```diff
@@ -116,25 +126,37 @@ event.emit('my_signal', {dummyKey: 'dummy_value'})
   })
 ```
 
+Catch `events` by subscribing to a Socket emmiting an event.
```
Comment (mattlsp) [inline docs/docs/building-sockets/core-api.md]: should be "emitting"

**Pull request merged**